### PR TITLE
Replacing "work anywhere" language

### PR DESCRIPTION
**Pull request by HilaryClarke**
Replaced previous "work anywhere" language with updated "work across the globe" language.

### CODE REVIEW

File: operations/workplace/people/working-at-mattermost/README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ description: A guide for Mattermost staff working anywhere
 
 # Working at Mattermost
 
-### **Work Anywhere**
+### **Work Across the Globe**
```
Comment (enahum) [inline operations/workplace/people/working-at-mattermost/README.md:7]: I just reas the title and the first thing that came to mind was “Would I have to work at more than one place?”

Just a thought from a non-native english speaker.

**Review by jasonblais — COMMENTED** [review]
@HilaryClarke Thanks for the pull request! Process-wise everything looks correct.

Please see the feedback from @enahum on how "work across the globe" may be interpreted.

File: operations/workplace/people/working-at-mattermost/README.md (hunk 1 above)
Comment (HilaryClarke) [reply operations/workplace/people/working-at-mattermost/README.md:7]: Great feedback. I'll ask this in the R&D leads channel to see how people react.

**Comment by HilaryClarke** [comment]
Thanks again @enahum. Given I didn't receive any additional feedback in channel, let's publish the proposed changes and we can iterate if needed in the future. 

Also, I just noticed that the description for this page is "A guide for Mattermost staff working anywhere."  Can you please change that to "A guide for Mattermost's distributed staff" ?

If there's a better system to propose new changes, let me know. I'm very new to Github. Thanks!

**Comment by enahum** [comment]
@HilaryClark the changes for the description can be done by you on this same PR, just edit the file here in the PR.

Happy to show you how it can be done

**Review by jasonblais — APPROVED** [review]
Approving as per notes above, thanks Hilary!

**Pull request merged**